mlflow/sagemaker/__init__.py

```diff
@@ -1336,6 +1336,19 @@ def _get_sagemaker_config_name(endpoint_name):
     return f"{endpoint_name}-config-{get_unique_resource_id()}"
 
 
+def _get_sagemaker_config_tags(endpoint_name):
+    return [{"Key": "app_name", "Value": endpoint_name}]
+
+def _add_sagemaker_tags(tags, config_tags):
```

Comment:
The args are little bit confusing, it's hard to tell the diff between `tags` and `config_tags`. We can either add a docstring to explain or use a more explicit name, e.g., `tags` and `app_name_tag`.